keystone-init/templates/_keystone_env.tpl

```diff
@@ -0,0 +1,57 @@
+{{- /* Generate a list of environment vars for Keystone Auth */}}
```

Comment:
aside: I think I'd like to use this template for all our keystone env vars. It will just grab an `auth:` block from values and convert it to standard `OS_` vars, and lets users reference secrets for any of the fields.